life_expectancy_gdp.ipynb

```diff
@@ -14,46 +21,116 @@
-      "6    Chile  2006                              78.9  1.547880e+11\n",
-      "7    Chile  2007                              78.9  1.736060e+11\n",
-      "8    Chile  2008                              79.6  1.796380e+11\n",
-      "9    Chile  2009                              79.3  1.723890e+11\n",
-      "10   Chile  2010                              79.1  2.185380e+11\n",
-      "11   Chile  2011                              79.8  2.522520e+11\n",
-      "12   Chile  2012                              79.9  2.671220e+11\n",
-      "13   Chile  2013                              80.1  2.783840e+11\n",
-      "14   Chile  2014                              80.3  2.609900e+11\n",
-      "15   Chile  2015                              80.5  2.425180e+11\n",
-      "16   China  2000                              71.7  1.211350e+12\n",
-      "17   China  2001                              72.2  1.339400e+12\n",
-      "18   China  2002                              72.7  1.470550e+12\n",
-      "19   China  2003                              73.1  1.660290e+12\n"
+      "  Country  Year  Life expectancy at birth (years)           GDP\n",
+      "0   Chile  2000                              77.3  7.786093e+10\n",
+      "1   Chile  2001                              77.3  7.097992e+10\n",
+      "2   Chile  2002                              77.8  6.973681e+10\n",
+      "3   Chile  2003                              77.9  7.564346e+10\n",
+      "4   Chile  2004                              78.0  9.921039e+10\n"
      ]
     }
    ],
    "source": [
     "#Import Data From csv Files\n",
     "gdp_data = pd.read_csv('all_data.csv')\n",
-    "print(gdp_data.head(20))"
+    "print(gdp_data.head())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CONVERT GDP INTO TRILLIONS\n",
+    "\n",
+    "In order to make data for accessible, convert the gdp data into trillions.\n",
+    "\n",
+    "Note: Trillions abbr. is \"T\".\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  Country  Year  Life expectancy at birth (years)           GDP  GDP(T)\n",
+      "0   Chile  2000                              77.3  7.786093e+10    7.79\n",
+      "1   Chile  2001                              77.3  7.097992e+10    7.10\n",
+      "2   Chile  2002                              77.8  6.973681e+10    6.97\n",
+      "3   Chile  2003                              77.9  7.564346e+10    7.56\n",
+      "4   Chile  2004                              78.0  9.921039e+10    9.92\n"
+     ]
+    }
+   ],
+   "source": [
+    "gdp_data['GDP(T)'] = np.round(gdp_data.GDP/10000000000,2)\n",
+    "print(gdp_data.head(5))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DROP THE ORIGINAL GDP COLUMN\n",
```

Comment:
The GDP conversion to trillions is useful for visualizing and presenting the data, but we need the complete GDP data for more precise calculations.